src/platform-includes/distributed-tracing/instrumentation/custom-instrumentation/javascript.mdx

```diff
@@ -0,0 +1,86 @@
+<Alert title="Note" level="note">
+
+Distributed tracing **works out of the box** if you have "Performance Monitoring" enable and use one of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`), or the [Next.JS SDK](/platforms/javascript/guides/nextjs/).
+
+If you are using one of the integrations of Frameworks listed above, **you do not need to do anything** for distributed tracing to work!
+
+You can stop reading this page now and continue building an amazing product.
+
+</Alert>
+
+Please read the note above. If you want to implement distributed tracing "by hand" two things need to be done:
+
+- When loading the page, extract and store incoming tracing information from HTML `meta` tags.
+- When doing outgoing requests, inject tracing information to the outgoing request.
+
+To learn more see also: <PlatformLink to="/distributed-tracing/#how-does-distributed-tracing-work-in-sentry">How Does Distributed Tracing Work in Sentry?</PlatformLink>
+
+## Extract Tracing Information From HTML `meta` Tags
+
+This code snipped assumes, that the backend that renders the HTML, injects tracing information into the HTML as `meta` tags.
+
+See for example [Inject Tracing Information Into Rendered HTML](/platforms/python/distributed-tracing/instrumentation/custom-instrumentation/#inject-tracing-information-into-rendered-html) to learn how your backend can inject this information.
+
+On `pageload` do the following:
+
+```javascript
+// Read meta tag values
```

Comment:
```suggestion
```javascript
import {
  baggageHeaderToDynamicSamplingContext,
  extractTraceparentData,
} from '@sentry/utils';

function getMetaContent(metaName: string): string | null {
  const metaTag = getDomElement(`meta[name=${metaName}]`);
  return metaTag ? metaTag.getAttribute('content') : null;
}

// Read meta tag values
```

We don't export `getMetaContent` from the SDK yet 😢

Reply:
we decided on not showing the custom instrumentation yet, we will revisit after tracing without performance was implemented in JS